src/main/java/qtlog/Main/Main.java

```diff
@@ -1,28 +1,57 @@
 package qtlog.Main;
 
+import java.util.Scanner;
+
 import qtlog.DPSReportController.LogUploader;
 import qtlog.DataModel.DataModel;
 import qtlog.DiscordController.DiscordController;
 import qtlog.FilesystemController.FileMonitor;
 import qtlog.UIController.UIController;
 import qtlog.UserInterface.UserInterface;
-import qtlog.shared.FileDTO;
+import qtlog.util.ConfigManager;
 
 public class Main {
     public static void main(String[] args) {
         start();
     }
 
     public static void start(){
+        //Get Log Folder Path on first startup 
+        String folderPath = ConfigManager.readLogPath();
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Idk, I think this is kinda pointless. I would just assume the default path and allow it to be overwritten in the config manually if needed. Forcing the user to enter the path is just worse experience in most cases. Same for the webhooks, these are supposed to be an optional step, so why force the user to set it for the program to function?

Reply:
same as above